### PR TITLE
Modify password store to avoid keyring prompts

### PR DESCRIPTION
**Pull request by helin24**
Since JxBrowser uses Chromium we have some situations where a user opens the embedded browser and is prompted to unlock the keyring in Linux if Chrome hasn't been opened already. We're hoping to avoid this by passing a Chrome option.

### CODE REVIEW

File: src/io/flutter/jxbrowser/EmbeddedBrowserEngine.java [hunk 1]
```diff
@@ -36,6 +37,8 @@ public EmbeddedBrowserEngine() {
     final EngineOptions options =
       EngineOptions.newBuilder(SystemInfo.isWindows ? OFF_SCREEN : HARDWARE_ACCELERATED)
         .userDataDir(Paths.get(dataPath))
+        .passwordStore(PasswordStore.BASIC)
+        .addSwitch("--use-mock-keychain")
```
Comment (vlad-lubenskyi) [inline src/io/flutter/jxbrowser/EmbeddedBrowserEngine.java:41]: This seems to be a macOS-specific switch and it won't help.

File: src/io/flutter/jxbrowser/EmbeddedBrowserEngine.java [hunk 2]
```diff
@@ -36,6 +37,8 @@ public EmbeddedBrowserEngine() {
     final EngineOptions options =
       EngineOptions.newBuilder(SystemInfo.isWindows ? OFF_SCREEN : HARDWARE_ACCELERATED)
         .userDataDir(Paths.get(dataPath))
+        .passwordStore(PasswordStore.BASIC)
```
Comment (vlad-lubenskyi) [inline src/io/flutter/jxbrowser/EmbeddedBrowserEngine.java:40]: When using `BASIC` password store, Chromium stores password without encryption and doesn't require access to the system keyring. I expect this option to help.

**Comment by helin24** [comment]
@vlad-lubenskyi thanks for the additional info!

**Review by stevemessick — APPROVED** [review]

**Review by jacob314 — COMMENTED** [review]
lgtm

**Pull request merged**